piqe_ocp_lib/tests/helpers.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import jmespath
+
+from typing import Optional, Tuple
+
+from piqe_ocp_lib.api.resources import OcpClusterVersion
+
+
+#  Change to dataclass (requires py >= 3.7)
+class Config:
+    def __init__(self, kubeconfig: Optional[str] = None):
+        self.kubeconfig = kubeconfig or os.environ.get("KUBECONFIG")
+
+    @property  # Change to cached_property (py >= 3.8)
+    def version(self) -> Tuple[int, int, int]:
```

Comment:
Might want to rename this property to cluster_version. My first reaction was to think that this is ocp version. Note that ocp_base has a 'version' property for the product version. Now that I think of it, are 'version' on ocp_base and what get_cluster_version redundant?

Reply:
Might be worth refactoring in favor of `OcpBase`